docs/guides/security/firewalls/how-to-install-bcc/index.md

```diff
@@ -0,0 +1,208 @@
+---
+slug: how-to-install-bcc
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: 'Learn to enhance the security of the server through the use of eBPF tracing tools'
```

Comment:
Needs and og_description as well (which can be the same as the normal description)